### ISSUE TITLE
Perhaps the following template might work? try adjusting it 

### ISSUE THREAD

**Issue by divingdiv** [HALAXAR/Face2Anime-project#1]
from PIL import Image

image_path = "path/to/your/image.jpg"  # Replace with your image path
img = Image.open(image_path).convert('RGB')  # Ensure RGB mode

import torchvision.transforms as transforms

transform = transforms.Compose([
    transforms.Resize((256, 256)),  # Adjust based on model input size
    transforms.ToTensor(),
    transforms.Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))  # Assuming PyTorch normalization
])

img_tensor = transform(img).unsqueeze(0)  # Add batch dimensionimport torch

class AnimeGenerator(torch.nn.Module):
    # ... (your model architecture definition)

class HumanGenerator(torch.nn.Module):
    # ... (your model architecture definition)

device = torch.device("cuda" if torch.cuda.is_available() else "cpu")  # Use GPU if available

anime_generator = AnimeGenerator().to(device)
anime_generator.load_state_dict(torch.load("./models/anime_generator.pth"))
anime_generator.eval()  # Set to evaluation mode

human_generator = HumanGenerator().to(device)
human_generator.load_state_dict(torch.load("./models/human_generator.pth"))
human_generator.eval()  # Set to evaluation mode

# Translate image to anime style using anime generator
with torch.no_grad():  # Disable gradient calculation for efficiency
    anime_img = anime_generator(img_tensor.to(device))

# Translate back to human style using human generator (optional)
if you_want_roundtrip_translation:
    with torch.no_grad():
        human_img = human_generator(anime_img.to(device))

def denormalize(img_tensor):
    mean = [0.5, 0.5, 0.5]
    std = [0.5, 0.5, 0.5]  # Adjust based on your normalization settings
    return img_tensor.mul(std).add(mean).clamp(0, 1)

anime_img = denormalize(anime_img.cpu().squeeze(0))  # Remove batch dimension
anime_img = transforms.ToPILImage()(anime_img)

if you_want_roundtrip_translation:
    human_img = denormalize(human_img.cpu().squeeze(0))
    human_img = transforms.ToPILImage()(human_img)

# Save translated images
anime_img.save("anime_translation.jpg")
if you_want_roundtrip_translation:
    human_img.save("human_translation.jpg")

**Comment by divingdiv** [comment]
https://stackoverflow.com/questions/51857274/how-can-i-load-and-use-a-pytorch-pth-tar-model

**Comment by divingdiv** [comment]
# to store
torch.save({
    'state_dict': model.state_dict(),
    'optimizer' : optimizer.state_dict(),
}, 'filename.pth.tar')

# to load
checkpoint = torch.load('filename.pth.tar')
model.load_state_dict(checkpoint['state_dict'])
optimizer.load_state_dict(checkpoint['optimizer'])

**Comment by divingdiv** [comment]
havent saved the model's state it seems.... {sigh}

**Comment by HALAXAR** [comment]
The issue has been addressed by introducing some preprocessing of input images
